File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    let n=x.toString();
    let m=n.length;
    for(let i=0;i<Math.floor(m/2);i++){
        if(n.charAt(i)!=n.charAt(m-i-1)) return false
    }
    return true
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        if(x<0):return False
        else:
            x=str(x)
            i=0
            j=len(x)-1
            print(i,j)
            while(i<j):
                if x[i]==x[j]:
                    i+=1
                    j-=1
                    continue
                else:
                    return False
            return True
        